airflow/bin/cli.py

```diff
@@ -54,6 +54,7 @@
                             Connection, DAG)
 
 from airflow.ti_deps.dep_context import (DepContext, SCHEDULER_DEPS)
+from airflow.utils import cli
```

Comment:
NIT: `import cli as cli_utils` would make it a bit more clear

Reply:
Will do.